### ISSUE TITLE
Feature Request: check app version when showing room ui.

### ISSUE THREAD

**Issue by stolk** [playgameservices/android-basic-samples#162]
It would be nice if Game Services would, when showing room UI, would also check the latest app version at Google Play, and if a newer is available, notify the user and trigger a download. This way, matches between players that are on different app versions would be less likely.

**Comment by claywilkinson** [comment]
The version of play services is checked when the the API Client is created.  You can read more about how it works at https://developers.google.com/android/guides/setup.

The relevant paragraph:

> You are strongly encouraged to use the GoogleApiClient class to access Google Play services
> features. This approach allows you to attach an OnConnectionFailedListener object to your client. To
> detect if the device has the appropriate version of the Google Play services APK, implement the onConnectionFailed() callback method. If the connection fails due to a missing or out-of-date version of the Google Play APK, the callback receives an error code such as SERVICE_MISSING, SERVICE_VERSION_UPDATE_REQUIRED, or SERVICE_DISABLED. To learn more about how to build your client and handle such connection errors, see Accessing Google APIs.

**Comment by stolk** [comment]
I was referring to my own app version, not the version of the Play Games app.
Many games that are multi player would like to ensure the participants in a match are running the latest game version. Instead of each game rolling its own solution to this, it would be nice if Game Services would provision for this.

**Comment by claywilkinson** [comment]
Ohh - so something softer than variant - so players can be matched, but then prompted to get the latest?  I'll pass the idea along!

**Comment by stolk** [comment]
Good point about 'variant', which would enable me to bar players with different game versions matched. I will use that. I would need to add a warning mechanism myself I guess, via a text file on a webserver or something.
